numpy/core/src/umath/loops.c.src

```diff
@@ -571,6 +571,22 @@ PyUFunc_O_O_method(char **args, npy_intp *dimensions, npy_intp *steps, void *fun
         PyObject **out = (PyObject **)op1;
         PyObject *ret = PyObject_CallMethod(in1 ? in1 : Py_None, meth, NULL);
         if (ret == NULL) {
```

Comment:
I think we need to catch the lookup error but not the call error - what you have right now will chain onto exceptions thrown by the method itself, and claim that the method didn't exist in the first place. So splitting PyObject_CallMethod into a separate lookup and call.

Reply:
makes sense